sdk/js/src/Client.ts

```diff
@@ -47,6 +47,11 @@ export class DVCClient implements Client {
             .then((config) => {
                 const oldConfig = this.config
                 this.config = config as BucketedUserConfig
+
+                if (options?.enableCloudData === true || this.config.project.settings?.cloudEntityData?.enabled) {
```

Comment:
you have to use the value of enableCloudData here if it's set. If its a boolean, use its value. If it isn't set, use the project setting